hugolib/hugo_modules_test.go

```diff
@@ -151,7 +151,7 @@ JS imported in module: |
         "devDependencies": {
                 "postcss-cli": "7.8.0",
                 "tailwindcss": "1.8.0"
- 
+
```

Comment:
Why the whitespace?

Reply:
Nevermind.  I see the empty space now that's I've commented.

Reply:
My IDE auto-removed whitespace at end-of-line in files that I touched.